src/network/core/address.cpp

```diff
@@ -411,3 +412,25 @@ void NetworkAddress::Listen(int socktype, SocketList *sockets)
 	getpeername(sock, (sockaddr *)&addr, &addr_len);
 	return NetworkAddress(addr, addr_len).GetAddressAsString();
 }
+
+/**
+ * Convert a string containing either "hostname", "hostname:port" or invite code
+ * to a ServerAddress, where the string can be postfixed with "#company" to
+ * indicate the requested company.
+ *
+ * @param connection_string The string to parse.
+ * @param default_port The default port to set port to if not in connection_string.
+ * @param company Pointer to the company variable to set iff indicted.
```

Comment:
s/indicted/indicated/ ? I highly doubt 'indicted' is the word you meant to use as it is only used in law settings.